src/rocq_elpi_vernacular_syntax.mlg

```diff
@@ -78,6 +78,19 @@ let lib_ref id =
           ++ str " (you may need to require some .v file with \
                   `Register ... as " ++ str id ++ str ".`).")
 
+let lookahead_lib_colon kwstate strm =
+  match Gramlib.LStream.peek_nth kwstate 0 strm with
+  | Tok.IDENT "lib" ->
+      (match Gramlib.LStream.peek_nth kwstate 1 strm with
+      | Tok.KEYWORD ":" -> ()
+      | _ -> raise Gramlib.Stream.Failure)
+  | _ -> raise Gramlib.Stream.Failure
```

Comment:
why not use the lookahead combinators?

Reply:
No special reason, I copied some code from SSR